.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "singleQuote": true
```

Comment:
Why not all the settings from https://github.com/storybooks/storybook/blob/master/.eslintrc.js#L38-L43 ?